packages/odf/components/create-storage-system/create-storage-system-steps/data-protection-step/data-protection-step.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { Form, Checkbox } from '@patternfly/react-core';
+import { WizardDispatch, WizardState } from '../../reducer';
+
+export const DataProtection: React.FC<DataProtectionProps> = ({
+  state,
```

Comment:
better to call it `dataProtection`, `state` represents entire wizrd state...
```suggestion
  dataProtection,
```